doc/man7/provider-base.pod

```diff
@@ -7,168 +7,186 @@ provider-base
 
 =head1 SYNOPSIS
 
- #include <openssl/core_numbers.h>
+ #include <openssl/core_dispatch.h>
 
  /*
-  * None of these are actual functions, but are displayed like this for
-  * the function signatures for functions that are offered as function
-  * pointers in OSSL_DISPATCH arrays.
+  * The following function prototypes don't represent actual C functions
+  * but rather the signatures of so-called 'dispatched' functions.
+  * To indicate the difference, curly {braces} have been added around
```

Comment:
I *like* that.  Wouldn't mind seeing that consistently in all provider pods.

Reply:
Is it ok if I split the pod changes into a separate pr? This would speed up the merging of this pr.

Reply:
I'm ok with that

Reply:
Ok, I dropped the experimental commit and rebased the pull request (without further changes) to the tip of master, just to make sure I didn't miss any new dispatched functions on master in the mean time. (This was not the case.)  Ready for review.